### PR TITLE
Ide 3594 fragment

### PR DESCRIPTION
**Pull request by yanan**

### CODE REVIEW

**Comment by yanan** [comment]
Run ok on Jenkins

File: ui-tests/tools/com.liferay.ide.ui.module/com.liferay.ide.ui.module.wizard/com.liferay.ide.ui.module.wizard.create/src/com/liferay/ide/ui/module/tests/NewComponentWizardMavenTests.java [hunk 1]
```diff
@@ -142,40 +147,16 @@ public void createComponentServiceWrapper() {
...
-
-		wizardAction.finish();
-
-		Assert.assertTrue(
-			viewAction.project.visibleFileTry(project.getName(), "src/main/java", packageName, className + ".java"));
-
-		viewAction.project.closeAndDelete(project.getName());
-	}
-
-	@Ignore("ignore as it may fails but happen unstable, need more research for it")
```
Comment (jtydhr88) [inline ui-tests/tools/com.liferay.ide.ui.module/com.liferay.ide.ui.module.wizard/com.liferay.ide.ui.module.wizard.create/src/com/liferay/ide/ui/module/tests/NewComponentWizardMavenTests.java]: how do we research on the unstable test?

File: ui-tests/tools/com.liferay.ide.ui.module/com.liferay.ide.ui.module.wizard/com.liferay.ide.ui.module.wizard.create/src/com/liferay/ide/ui/module/tests/NewComponentWizardMavenTests.java [hunk 2]
```diff
@@ -53,76 +79,55 @@ public void createComponentModelListener() {
...
 
 		Assert.assertTrue(
-			viewAction.project.visibleFileTry(project.getName(), "src/main/java", packageName, className + ".java"));
+			viewAction.project.visibleFileTry(
+				project.getName(), "src/main/java", packageName, className + ".java [Component]"));
 
 		viewAction.project.closeAndDelete(project.getName());
 	}
 
-	@Ignore("ignore as it may fails but happen unstable, need more research for it")
```
Comment (jtydhr88) [inline ui-tests/tools/com.liferay.ide.ui.module/com.liferay.ide.ui.module.wizard/com.liferay.ide.ui.module.wizard.create/src/com/liferay/ide/ui/module/tests/NewComponentWizardMavenTests.java]: how do we research on the unstable test?

File: ui-tests/tools/com.liferay.ide.ui.module/com.liferay.ide.ui.module.wizard/com.liferay.ide.ui.module.wizard.create/src/com/liferay/ide/ui/module/tests/NewComponentWizardMavenTests.java [hunk 3]
```diff
@@ -28,7 +29,32 @@
  */
 public class NewComponentWizardMavenTests extends SwtbotBase {
 
-	@Ignore("ignore as it may fails but happen unstable, need more research for it")
```
Comment (jtydhr88) [inline ui-tests/tools/com.liferay.ide.ui.module/com.liferay.ide.ui.module.wizard/com.liferay.ide.ui.module.wizard.create/src/com/liferay/ide/ui/module/tests/NewComponentWizardMavenTests.java]: how do we research on the unstable test?

**Comment by jtydhr88** [comment]
hey @yanan 
lets open diffenert PRs, one for new tests, one for re-enable ignore test

**Comment by yanan** [comment]
Got it

**Pull request closed without merging**